### ISSUE TITLE
Add thread level options

### ISSUE THREAD

**Issue by huynhsontung** [huynhsontung/Indirect#19]
Ability to mute thread, delete thread

**Comment by MarcAnt01** [comment]
Yeah, was about to open a ticket about it. I was thinking about a flyout menu in chats list to open with right click/long press
